prime-router/src/main/kotlin/Report.kt

```diff
@@ -1542,7 +1542,7 @@ class Report : Logging {
                         ?: "${Schema.formBaseName(schemaName)}-$id-${formatter.format(createdDateTime)}"
                 }
             }
-            return "$fileName.$nameSuffix"
+            return "$fileName.$nameSuffix".trimStart('-')
```

Comment:
Nit: maybe a test to cover this edge case?

Reply:
Removed this change since now schemaName should always be provided (None in the case of passthrough).